src/v/cluster/partition_balancer_backend.cc

```diff
@@ -522,6 +522,7 @@ partition_balancer_overview_reply partition_balancer_backend::overview() const {
     if (!_cur_term || _raft0->term() != _cur_term->id) {
         // we haven't done a single tick in this term yet, return empty response
         ret.status = partition_balancer_status::starting;
+        ret.partitions_pending_force_recovery_count = -1;
```

Comment:
Hmm, why -1? Isn't 0 semantically correct?

Reply:
We can have `status != "starting" and force_recovery_count == 0` in the test predicate

Reply:
Don't we need to differentiate between not having the right count vs no partitions to force recover?

Reply:
So it can be non-zero, we just didn't have a chance to update it yet? I guess -1 makes sense then, thanks.